### ISSUE TITLE
SMPLer-X: Scaling Up Expressive Human Pose and Shape Estimation

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#24]
1.どんな論文? /What does this work do?

- https://arxiv.org/abs/2309.17448
- https://caizhongang.com/projects/SMPLer-X/
- NeurIPS 2023
- A Expressive Human Pose and Shape estimation (EHPS) model named SMPLer-X, which exhibits strong performance across diverse test benchmarks and excellent transferability to even unseen environments.
- Build a systematic and comprehensive benchmark on EHPS datasets, which provides critical guidance for scaling up the training data toward robust and transferable EHPS.

2.先行研究と比べてどこがすごい? /What makes this work greater than existing works?

- SMPLer-X training with up to 4.5M instances from diverse data sources, rather than a confined set of training datasets( as compared to other current SOTA methods).

- SMPLer-X delivers balanced results across various scenarios and extends successfully to unseen datasets, its foundation model consistently delivers state-of-the-art results on seven benchmarks such as AGORA (107.2 mm NMVE), UBody (57.4 mm PVE), EgoBody (63.6 mm PVE), and EHF (62.3 mm PVE without finetuning).

- SMPLer-X is sensitive to Scaling up, both data and model scaling are effective in reducing mean errors on primary metrics across key benchmarks.
![FIG1](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/c9e0d27c-ba32-4c71-9e61-cf28ae6ce700)
> This conclusion holds for the AGORA, UBody, EgoBody, 3DPW and EHF datasets. OSX and H4W are SOTA methods. Area of the circle indicates model size, with ViT variants as the reference (top right).

3.技術や手法のキモはどこ? /What is the heart of this technology or method?

SMPLer-X consists of three parts: **Backbone**, **Neck**, and **Head**.
![19ec1d14f0035026a04319b4bf4cb01](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/d6222aca-6d29-44d9-8cd0-8633d205f952)
> **Backbone:** extracts image features, which we employ Vision Transformer for its scalability;
> **Neck:** predicts bounding boxes and crop regions of interest from the feature map for hands and face; 
> **Head:** regression heads that estimate parameters for each part.

4.どうやって有効だと検証した? /How is this work validated?

**Quantitative results:** Authors show detailed by-part results of body, hands, and face on main benchmarks such as AGORA test set (Table 3), AGORA validation set (Table 4), UBody (Table 6), EgoBody-EgoSet (Table 7) and EHF (Table 5).
![tab3-5](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/eb3c9e3e-6364-4fae-b59a-abebbf2d0f5c)
![tab6,7](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/8cc488e6-1c85-4a5a-aea4-1ae9f80c354d)

Furthermore, Authors evaluate the transferability of our foundation models on two more benchmarks: ARCTIC (Table 8) and DNA-Rendering (Table 9).
![tab8,9](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/f82a53e7-916e-4ced-9a81-0695250e747d)

**Ablation studies:** Authors did the ablation study on three different sampling strategies, including: **Balanced**, **Weighted**, and **Concat**. 
![31fa0ae2896a91ff9883a4354443747](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/179db984-9354-4bb6-89b4-29f90b91c65d)
The performance of the foundation model is not sensitive to the sampling strategy as shown in Table 13.

5.議論はある? /Any discussion for this work?

6.読んでいてわからなかったところは? /What don't you understand for this paper?

まだ十分に理解できていないので、後でSMPLer-Xについて調べていく必要があります。

7.公開コードやデータセットは? /Public codes or datasets?

https://github.com/caizhongang/SMPLer-X

8.次に読むべき論文は? /What should we read next
